relocations/quarkus-test-vault/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-relocations-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-test-vault</artifactId>
+
+    <distributionManagement>
+        <relocation>
+            <groupId>io.quarkiverse.vault</groupId>
+            <artifactId>quarkus-test-vault</artifactId>
+            <version>1.0.0</version>
+            <message>The artifact ${project.artifactId} has been moved to io.quarkiverse.vault. Please update your dependency.</message>
```

Comment:
Same here.

Reply:
Ah shoot, yeah, they didn't show with autocomplete. I will fix it but I'll let CI finish then fix them and merge if it's good for you?

Reply:
No problem